pkg/aws/eni/node.go

```diff
@@ -596,9 +596,6 @@ var adviseOperatorFlagOnce sync.Once
 // GetMinimumAllocatableIPv4 returns the minimum amount of IPv4 addresses that
 // must be allocated to the instance.
 func (n *Node) GetMinimumAllocatableIPv4() int {
-	n.mutex.RLock()
-	defer n.mutex.RUnlock()
-
```

Comment:
What happens to parallel access to these functions now? Do we assume that the lock is taken?